examples/nodejs/gcFormsApiClient.ts

```diff
@@ -0,0 +1,69 @@
+import axios, { type AxiosInstance } from "axios";
+import type {
+  EncryptedFormSubmission,
+  FormTemplate,
+  NewFormSubmission,
+} from "./dataStructures.js";
+
+export class GCFormsApiClient {
+  private httpClient: AxiosInstance;
+
+  public constructor(apiUrl: string, accessToken: string) {
+    this.httpClient = axios.create({
```

Comment:
Nice use of creating a base config for axios and reusing it across the function calls